rasa/shared/importers/rasa.py

```diff
@@ -65,7 +65,6 @@ async def get_domain(self) -> Domain:
             return domain
         try:
             domain = Domain.load(self._domain_path)
-            domain.check_missing_templates()
```

Comment:
Shouldn't this check be done somewhere if not here?

Reply:
I've added it [here](https://github.com/RasaHQ/rasa/pull/7390/files#diff-663c0521028355e3657687cc698278f0d680619c6897bbe0762fe0e40213143eR328)